serialize.go

```diff
@@ -0,0 +1,132 @@
+package cronrange
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var (
+	strSingleWhitespace = ` `
+	strDoubleQuotation  = `"`
+	strSemicolon        = `;`
+	strMarkDuration     = `DR=`
+	strMarkTimeZone     = `TZ=`
+
+	errIncompleteExpr     = errors.New("expression should contain at least two parts")
+	errMissDurationExpr   = errors.New("duration is missing from the expression")
+	errEmptyExpr          = errors.New("expression is empty")
+	errJsonNoQuotationFix = errors.New(`json string should start and end with '"'`)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [var errJsonNoQuotationFix should be errJSONNoQuotationFix](https://app.codacy.com/app/an9an63/cronrange/pullRequest?prid=4472251)